view/frontend/web/js/view/payment/method-renderer/adyen-oneclick-method.js

```diff
@@ -257,30 +259,14 @@ define(
                             }
                             var oneClickCardNode = document.getElementById('cvcContainer-' + self.value);
 
-                            // this should be fixed in new version of checkout card component
-                            var hideCVC = false;
```

Comment:
don't we still need this for RECURRING billing agreements?

Reply:
Thanks will investigate if the hideCVC still is possible in the new component.

Reply:
the component now supposed to handle it but good to verify it on the e2e instance when uploaded.

Reply:
valid point will revert this piece of code.

Reply:
yeah indeed wondering how this works now.

Reply:
@cyattilakiss tests are successful, are we testing this setup though with recurring contracts only ?

Reply:
I could revert it to: 
 if (!this.hasVerification()) {
 hideCVC = true;
}
just to cover this scenario. The maestro fix here is not needed anymore that for sure.
@Aleffio ok with this approach ?

Reply:
👍

Reply:
@rikterbeek we only test with one click - that should be the default setup for the e2e instance

Reply:
done.